src/index.html

```diff
@@ -17,6 +29,82 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
```

Comment:
Add an empty line between the `<a>` and `<nav>` tags to separate sibling blocks.